dspace-server-webapp/src/test/java/org/dspace/app/rest/test/AbstractControllerIntegrationTest.java

```diff
@@ -80,7 +79,7 @@
     public static final String BASE_REST_SERVER_URL = "http://localhost";
 
     protected MediaType contentType = new MediaType(MediaTypes.HAL_JSON.getType(),
-                                                    MediaTypes.HAL_JSON.getSubtype(), StandardCharsets.UTF_8);
```

Comment:
Was there a specific reason to remove the "StandardCharsets.UTF_8" here ? Seems kind of strange to remove it.

Reply:
@KevinVdV : Yes, this was required to be removed by Spring v5.2.4 upgrade.  Please notice the comment in the description above where I said:

> This includes the removal of `MediaType.APPLICATION_JSON_UTF8` (in favor of just `MediaType.APPLICATION_JSON`), as JSON best practices are now to only have a generic media type. See https://github.com/spring-projects/spring-framework/issues/22788#issuecomment-486568650

Because JSON best practices are to NOT include a character set in JSON files, Spring now no longer supports JSON files that claim to be UTF-8 (i.e. `MediaType.APPLICATION_JSON_UTF8`), instead requires the more generic `MediaType.Application_JSON`.

So, yes, this looks odd. But Spring Boot & Spring no longer support the more specific UTF-8 JSON media type. They just support the generic JSON media type.

Reply:
Ok great, makes sense ! Just wanted to make sure this wasn't an oversight.